<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesWindow</class>
 <widget class="QDialog" name="PreferencesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lbl_pref">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Wizard Preferences</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbx_genPrefs">
     <property name="title">
      <string>Usage Preferences</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="cbx_checkMayaDir">
        <property name="text">
         <string>Check Maya directory</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="cbx_createCMake">
        <property name="text">
         <string>Create CMakeList.txt as default</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="cbx_createQt">
        <property name="text">
         <string>Create Qt Project as default</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="cbx_createNewDir">
        <property name="text">
         <string>Create new directory as default</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbx_paths">
     <property name="title">
      <string>Default Paths</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QLineEdit" name="line_genDir"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_defaultGenDir">
        <property name="text">
         <string>Default generation directory</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="line_vendor"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="line_mayaRootDir"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_mayaDir">
        <property name="text">
         <string>Maya root directory</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbl_defaultVendor">
        <property name="text">
         <string>Default vendor</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btn_browseForMayaRootDirectory">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btn_browseForGenDir">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_save">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbx_checkMayaDir</tabstop>
  <tabstop>cbx_createQt</tabstop>
  <tabstop>cbx_createCMake</tabstop>
  <tabstop>cbx_createNewDir</tabstop>
  <tabstop>line_mayaRootDir</tabstop>
  <tabstop>btn_browseForMayaRootDirectory</tabstop>
  <tabstop>line_genDir</tabstop>
  <tabstop>btn_browseForGenDir</tabstop>
  <tabstop>line_vendor</tabstop>
  <tabstop>btn_save</tabstop>
  <tabstop>btn_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
